strawberryfields/gbs/sample.py

```diff
@@ -142,6 +151,10 @@ def sample(A: np.ndarray, n_mean: float, n_samples: int = 1, threshold: bool = T
     with p.context as q:
         sf.ops.GraphEmbed(A, mean_photon_per_mode=mean_photon_per_mode) | q
 
+        if loss:
```

Comment:
seems like loss is a Boolean here but a float elsewhere.

Reply:
By the way, applying loss in the Gaussian backend is very cheap so you loose very little by always applying it.

Reply:
Thanks Nico! Actually, `if` evaluates to `True` for any non-zero float, and is `False` if the float is zero

Reply:
I always though Python got Booleans precisely to not do that, like in C. Anyway, if that is how things work, then it's fine.